mail_activity_reminder/models/mail_activity.py

```diff
@@ -111,25 +111,43 @@ def action_remind(self):
         """
         Group reminders by user and type and send them together
         """
-        MailThread = self.env["mail.thread"]
         utc_now = fields.Datetime.now().replace(tzinfo=UTC)
         for user in self.mapped("user_id"):
             activities = self.filtered(lambda activity: activity.user_id == user)
             tz = timezone(user.sudo().tz or "UTC")
             local_now = utc_now.astimezone(tz)
+            # Send reminders per dedicated mail templates if any
+            mail_templates = activities.activity_type_id.reminder_mail_template_id
```

Comment:
wondering: wouldn't be better to set the default template as a default for the field and simplify the logic?